src/EditorFeatures/Test/DocCommentFormatting/DocCommentFormattingTests.cs

```diff
@@ -50,7 +50,7 @@ public void ExampleAndCodeTags()
             results in <c>p</c>'s having the value (2,8).
             </example>";
 
-            var expected = "This method changes the point's location by the given x- and y-offsets. For example:\r\n\r\nPoint p = new Point(3,5); p.Translate(-1,3);\r\n\r\nresults in p's having the value (2,8).";
+            var expected = "This method changes the point's location by the given x- and y-offsets. For example:\r\n\r\n\r\n            Point p = new Point(3,5);\r\n            p.Translate(-1,3);\r\n            \r\n\r\nresults in p's having the value (2,8).";
```

Comment:
📝 This is one of the cases that made me uncomfortable, but I'll reserve judgement until I see the new builder tests.

Reply:
@sharwell I'm expecting you'll still be uncomfortable after seeing the new tests 😄 
Does it make sense to trim line-endings from code elements?

Reply:
I'm going to look at screenshots before requesting changes to the actual normalization approach.